_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+---
+id: be4d0c
+name: Inline link has either distinguishable style not based on color (hue) alone or content that indicates it is a link
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style not based on color (hue) alone or have content that indicates they are links.
+accessibility_requirements:
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
```

Comment:
I am seeing this style of authoring applicability for the first time, where we prefix a line item with something like a definition per se:

- def 1: description. 

It kind of implies to me that the description is secondary & the sentence does not flow right.

```suggestion
- the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
```

Simply removing the prefix makes it better in my opinion.

Reply:
I'm just following the rule design instructions for listed conditions: https://act-rules.github.io/pages/design/rule-design/#listed-conditions

They apply to lists in both applicability and expectations. I guess we haven't had rules with lists of conditions in the applicability recently. 

I'm keeping this unless you really want me to change it.